src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -154,7 +154,9 @@ D2D1_RECT_F D2DOverlaySVG::get_snap_right() const {
 }
 
 
-D2DOverlayWindow::D2DOverlayWindow() : total_screen({}), animation(0.3) {
+
+D2DOverlayWindow::D2DOverlayWindow() : total_screen({}), anim_time(0.3) {
```

Comment:
Please rename `anim_time` to more descriptive `animation_time` or something similar.